content/how-to/add-code-servers/azure-devops/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Adding an Azure DevOps code server"
-description: "How to add an Azure DevOps code server to your tenant"
-lead: "How to add an Azure DevOps code server to your tenant"
```

Comment:
Actually, this was a good lead and description. They're different from the title and they accurately describe the page. I wouldn't remove these.